html/js/guiutils/Tools.js

```diff
@@ -54,13 +54,36 @@ GTE.UI = (function (parentModule) {
             case GTE.MODES.PLAYER_ASSIGNMENT:
                 buttonToSwitch = "button-player-" + this.activePlayer;
                 break;
+            case GTE.MODES.MERGE:
+                if (this.ableToSwitchToISetMode()) {
+                    buttonToSwitch = "button-merge";
+                    // If iset tools have never been chosen
+                    if (!this.isetToolsRan) {
+                        // Assign singleton isets to each node with no iset
+                        GTE.tree.initializeISets();
```

Comment:
This may be perfectly right, but I dont have time to check: is there any potential issue in the following scenario?

1. Create some nodes, assign them players
2. Press merge button (and initializeIsets gets called)
3. Create more nodes, assign them players
4. Press merge button again (and initializeISets doesnt get called this time, so maybe the new nodes dont have isets??)